internal/service/flink/flink_table.go

```diff
@@ -197,8 +197,9 @@ func ResourceFlinkTable() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
-		Schema:        aivenFlinkTableSchema,
-		CustomizeDiff: customdiff.If(schemautil.ResourceShouldExist, resourceFlinkTableCustomizeDiff),
+		Schema:             aivenFlinkTableSchema,
+		CustomizeDiff:      customdiff.If(schemautil.ResourceShouldExist, resourceFlinkTableCustomizeDiff),
+		DeprecationMessage: "This resource is deprecated and will be removed in the next major release.",
```

Comment:
Let's save it as a constant and use it in other resources too.

Reply:
done